### PR TITLE
Updates for package builds

### PR DESCRIPTION
**Pull request by bphinz**

### CODE REVIEW

File: contrib/packages/deb/ubuntu-noble/debian/control [hunk 1]
```diff
@@ -0,0 +1,193 @@
+Source: tigervnc
+Section: x11
+Priority: optional
+Maintainer: Brian P. Hinz <bphinz@users.sourceforge.net>
+Standards-Version: 3.8.4
+Build-Depends:
+# TigerVNC dependencies
```
Comment (CendioOssman) [inline contrib/packages/deb/ubuntu-noble/debian/control:7]: I see you did some cleanups in this list. Perhaps best to do in the other variants as well to make things easy to keep in sync?

On that note, gcrypt is not that commonly used any more. Are we sure Ubuntu still uses that for Xorg?

File: contrib/packages/deb/ubuntu-noble/debian/copyright [hunk 2]
```diff
@@ -0,0 +1,116 @@
+This package was packaged for Debian by Brian P. Hinz <bphinz@users.sourceforge.net>
+on Tue, 18 Jun 2024 18:54:03 -0400 using the tightvnc package as a base.
```
Comment (CendioOssman) [inline contrib/packages/deb/ubuntu-noble/debian/copyright:2]: I hadn't noticed this file before. We're obviously not keeping this copy in sync properly. Can we automate that? Or better yet, just put a notice here to check the README?

**Review by CendioOssman — APPROVED** [review]
Looks good to me! Just squash the commits and it should be good to merge.

I made some notes of general improvements we might consider for all Debian packages.

File: contrib/packages/deb/ubuntu-noble/debian/control (hunk 1 above)
Comment (bphinz) [reply contrib/packages/deb/ubuntu-noble/debian/control:7]: libgcrypt is still listed as a build requirement in Ubuntu's xorg-server package

File: contrib/packages/deb/ubuntu-noble/debian/copyright (hunk 2 above)
Comment (bphinz) [reply contrib/packages/deb/ubuntu-noble/debian/copyright:2]: I don't know if there is an automated way to keep it in sync but I will take a look at what debian's packaging guidelines are regarding this file.  For now I've updated them all manually.

**Review by CendioOssman — CHANGES_REQUESTED** [review]
I think something went wrong here. The merge should not be in to this branch.

**Review by CendioOssman — APPROVED** [review]

**Comment by CendioOssman** [comment]
I would suggest resolving it manually on the command line as part of merging to `master`. Or rebase your original commit after bionic was removed.

**Pull request merged**